docs/src/whatsnew/latest.rst

```diff
@@ -80,15 +85,20 @@ This document explains the changes made to Iris for this release
 🔗 Dependencies
 ===============
 
-#. N/A
-
+#. `@hsteptoe <https://github.com/hsteptoe>`_ added `rasterio <https://rasterio.readthedocs.io/en/stable/index.html>`_ 
+   and `affiine <https://affine.readthedocs.io/en/latest/>`_ as a dependency for :func:`iris.util.mask_cube_from_shapefile`. 
+   This is to support the new functionality of handling additional shapefiles and projections. (:issue:`6126`, :pull:`6129`)
```

Comment:
Very important to emphasise that this is an _optional_ dependency.

Reply:
Done